mf4-csv.py

```diff
@@ -44,22 +46,12 @@
 }
 
 def main():
```

Comment:
Isn't main normally at the bottom of the file, right above `if __name__ == '__main__'`? Also, why are you passing `1` to `file_convert`?

Reply:
Typo on the file convert. As per the placement of main, there is nothing that says it must be above the main guard but WE can make that a design requirement.